src/fork_gc.c

```diff
@@ -204,11 +204,8 @@ static bool FGC_childRepairInvidx(ForkGC *gc, RedisSearchCtx *sctx, InvertedInde
     // Capture the pointer address before the block is cleared; otherwise
     // the pointer might be freed! (IndexBlock_Repair rewrites blk->buf if there were repairs)
     void *bufptr = blk->buf.data;
-    int nrepaired = IndexBlock_Repair(blk, &sctx->spec->docs, idx->flags, params);
-    // We couldn't repair the block - return 0
-    if (nrepaired == -1) {
-      goto done;
-    } else if (nrepaired == 0) {
+    size_t nrepaired = IndexBlock_Repair(blk, &sctx->spec->docs, idx->flags, params);
+    if (nrepaired == 0) {
```

Comment:
not covered because it's from child?

Reply:
Correct, it is only used by the fork GC child, for which we don't collect coverage reports of today